core/bounds.gms

```diff
@@ -519,5 +519,12 @@ $IFTHEN.transpGDPscale not "%cm_transpGDPscale%" == "on"
   vm_transpGDPscale.fx(t,regi) = 1;
 $ENDIF.transpGDPscale
 
+***----------------------------------------------------------------------------
+*'  Limit slack variable and uncontrolled variable values for adj costs that limit changes to REF in cm_startyear
```

Comment:
could you change "REF" to "reference" to avoid people confusing this with the REF region?

Reply:
did so